src/libYARP_dev/CMakeLists.txt

```diff
@@ -84,7 +84,6 @@ set(YARP_dev_HDRS include/yarp/dev/all.h
                   include/yarp/dev/ServerSerial.h
                   include/yarp/dev/ServerSoundGrabber.h
                   include/yarp/dev/ServiceInterfaces.h
-                  include/yarp/dev/TestFrameGrabber.h
```

Comment:
We are not REmoving it, just moving it  :-) 
It has been transformed into a proper plugin. It is still used for testing purpose.